deployment/template-cronjob.yaml

```diff
@@ -154,4 +154,4 @@ parameters:
   required: true
   name: CRON_SCHEDULE
   # Run every monday 1.am
-  value: "0 1 * * MON"
+  value: "30 * * * *"
```

Comment:
Rather than changing those parameter here, can we change it to Sass Analytics repo? and create a dummy PR, what do you think?

Reply:
This is just for debugging purpose. saas-analytics process would need 2 PR & 2 reverts.